git-ftp

```diff
@@ -1519,7 +1580,17 @@ do
 			;;
 		*)
 			# Pass thru anything that may be meant for fetch.
-			[ -n "$1" ] && URL=$1
+            if [[ "$ACTION" == "clone" ]];then
```

Comment:
This seems a bit complicated to me. What do you think about this?
```bash
if [ -n "$1" ]; then
    if [ -n "$URL" ]; then
        URL="$1"
    elif [ "$ACTION" == "clone" -a -n "$CLONE_DIR" ]; then
        CLONE_DIR="$1"
    else
        print_error_and_die "Unrecognised option: $1"
    fi
fi
```
That should allow mixed options like: `git ftp clone -v ftp://example.com/ -u eddy ProjectFolder`